README.md

```diff
@@ -14,8 +14,57 @@ the packages individually, for example:
     make -C debs/xenial/archivematica
     make -C debs/xenial/archivematica-storage-service
 
-### Test packages
+#### Using parameters
 
+Most makefiles support parameters in the form of environment variables. They 
+may be a bit different between packages, but the most common are ``BRANCH``,
+``VERSION`` and ``RELEASE``.
+
+So in order to build a specific branch / version, this command can be used from
```

Comment:
Is the slash used to indicate the word *Or*? I think typographically speaking it should read "branch/version" instead.